### PR TITLE
remove duplicate rules from .eslintrc (close #292)

### LINKED ISSUE #292: We have duplicate rules in .eslintrc
And for some reason we have some important rules like `no-shadow` disabled.

### PR DESCRIPTION
**Pull request by miherlosev**
/cc @inikulin @churkin

### CODE REVIEW

File: src/client/sandbox/event/listeners.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ export default class Listeners extends EventEmitter {
     }
 
     _createEventHandler () {
-        var listeningCtx = this.listeningCtx;
+        var self = this;
```
Comment (inikulin) [inline src/client/sandbox/event/listeners.js]: Why we need this change? And BTW, never use `self`, `_this` or anything like that for the closures. Use meaningful names instead. E.g. in this case:
```js
var listeners = this;
```

File: src/client/sandbox/xhr/xml-http-request-wrapper.js [hunk 2]
```diff
@@ -1,64 +1,63 @@
...
+                    })(value);
+                }
+                else
+                    xhr[propName] = value;
 
+                return xhr[propName];
+            }
+        });
+    }
+    static _wrapFunc (xhr, xhrWrapper, funcName) {
```
Comment (inikulin) [inline src/client/sandbox/xhr/xml-http-request-wrapper.js]: Need new line here

**Comment by inikulin** [comment]
\r-

**Comment by miherlosev** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

File: src/client/sandbox/event/listeners.js [hunk 3]
```diff
@@ -68,7 +68,7 @@ export default class Listeners extends EventEmitter {
     }
 
     _createEventHandler () {
-        var listeningCtx = this.listeningCtx;
+        var listeners = this;
 
         return function (e) {
```
Comment (churkin) [inline src/client/sandbox/event/listeners.js]: We can replace it to the arrow

Comment (inikulin) [inline src/client/sandbox/event/listeners.js]: We can not: https://github.com/miherlosev/testcafe-hammerhead/blob/i_292/src/client/sandbox/event/listeners.js#L85

Comment (churkin) [inline src/client/sandbox/event/listeners.js]: Sorry

Comment (miherlosev) [inline src/client/sandbox/event/listeners.js]: I removed `eslint-disable comment` in two places:
https://github.com/miherlosev/testcafe-hammerhead/blob/i_292/src/client/sandbox/code-instrumentation/properties/index.js#L448
https://github.com/miherlosev/testcafe-hammerhead/blob/i_292/src/client/sandbox/code-instrumentation/properties/index.js#L454

**Comment by miherlosev** [comment]
FPR

**Comment by churkin** [comment]
lgtm

**Pull request merged**